warehouse/admin.go

```diff
@@ -77,30 +88,34 @@ func (wh *WarehouseAdmin) Query(s QueryInput, reply *warehouseutils.QueryResult)
 	return err
 }
 
-func (wh *WarehouseAdmin) QueryWhUploads(uploadsReq UploadsReqT, reply *[]byte) error {
-	uploadsReq.API = UploadAPI
-	res, err := uploadsReq.GetWhUploads()
-	if err != nil {
-		return err
+// Test the underlying warehouse
+func (wh *WarehouseAdmin) Test(s TestInput, reply *TestResponse) (err error) {
+	if strings.TrimSpace(s.DestID) == "" {
+		return errors.New("please specify the destination ID to query the warehouse")
 	}
-	bytes, err := json.Marshal(res)
-	if err != nil {
-		return err
+
+	var warehouse warehouseutils.WarehouseT
+	srcMap, ok := connectionsMap[s.DestID]
+	if !ok {
+		return errors.New("please specify a valid and existing destination ID")
 	}
-	*reply = bytes
-	return nil
-}
 
-func (wh *WarehouseAdmin) QueryWhTables(tableUploadReq TableUploadReqT, reply *[]byte) error {
-	tableUploadReq.API = UploadAPI
-	res, err := tableUploadReq.GetWhTableUploads()
-	if err != nil {
-		return err
+	for _, v := range srcMap {
+		warehouse = v
+		break
 	}
-	bytes, err := json.Marshal(res)
+
+	pkgLogger.Infof(`[WH Admin]: Test warehouse: %s:%s`, warehouse.Type, warehouse.Destination.ID)
```

Comment:
Can we update the log to be `pkgLogger.Infof([WH Admin]: Validating warehouse destination: %s:%s`